pilot/pkg/model/network.go

```diff
@@ -46,9 +47,15 @@ type NetworkGateway struct {
 	Port uint32
 }
 
-type NetworkGatewaysWatcher interface {
+type DefaultNetwork struct {
+	Network network.ID
+	Cluster cluster.ID
+}
+
+type NetworkConfigWatcher interface {
 	NetworkGateways() []NetworkGateway
 	AppendNetworkGatewayHandler(h func())
+	DefaultNetworks() []DefaultNetwork
```

Comment:
why do we need a slice

Reply:
Since the aggregated controller obtains default network information from different clusters, we need to use a slice. Or do you think a map would be more appropriate?